translations/de.json

```diff
@@ -12,7 +12,7 @@
     "ambrosia": {
         "amount": "You have <<var(--lightgoldenrodyellow-text-color)|{{ambrosia}}>> Ambrosia!",
         "blueberryGeneration": "Each second, gain <<var(--amber-text-color)|{{chance}}>> seconds of Blueberry Time.",
-        "perGen": "Currently gaining <<var(--amber-text-color)|{{guaranteed}}>> with a <<var(--amber-text-color)|{{extraChance}}%>> chance of another. [<<var(--lime-text-color)|Luck: &#9752 {{ambrosiaLuck}}>>]",
```

Comment:
I'm not sure about these changes. I assume crowdin will revert them.

Reply:
The clover change was made due to the way Perk Descriptions are handled, it does not properly read out the Decimal code and instead displays it raw. 

For perks it will be a required change in order for the text to display properly, I made the change globally to help conform it across the board and due to it being easier to read than the raw decimal code itself. I do not believe there should be any negative effects from this.

Reply:
It's unlikely that Platonic added these in himself, which means crowdin likely did so.

Reply:
I'll throw them in for now and if anything besides the perk text gets changed then it'll be no big deal.